### ISSUE TITLE
Don't force links to open in a new tab (Remove target=_blank from links)

### ISSUE THREAD

**Issue by tofumatt** [google/site-kit-wp#1684]
## Feature Description

We currently have a lot of links that open in new windows/tabs by default. An example: https://github.com/google/site-kit-wp/pull/1679#discussion_r441506247

Opening links in a new tab/window presents accessibility and usability problems. My main issue with it is that it removes user choice in how a link should be handled; I can choose to open a normal link in a new window or not, but if `target="_blank"` is set I have no option but to open the link in a new tab.

The main reason to use this feature legitimately is that there is user-input/flow we want to persist even if the link is opened, but in our case we should fix this by saving user state and allowing them to navigate away and then back (and pick up where they left off).

If there are places where opening links in a new tab is significantly easier than state persistence, then I think it's okay to split this into two issues:

1. remove all "easy win" new tab links
2. address ones that open in new tabs not to lose user input separately

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Acceptance criteria

* Links do not open in a new tab by default.

## Implementation Brief

* 

## QA Brief

* Ensure no links open in a new tab without the user explicitly opting to open them in a new tab/window

## Changelog entry

*

**Comment by felixarntz** [comment]
After several discussions about this, we're not going to change the behavior here. External links should always be marked as such, both with the external icon as well as with screen reader text; however completely avoiding external links is a non-goal, as e.g. the "Learn more" links, which are one of the primary surfaces for external links, shouldn't take the user away from Site Kit into a completely different area.
